### DISCUSSION TITLE
Bug? Can't switch OBS scenes using M³ bottom shortcuts

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2328] [General]
I've set up additional scenes in OBS (for alternative conductor video positions on top of an image).

After that, I can see a few shortcuts for these scenes in M³'s "Meeting mode". 

The problem:
I can easily switch these scenes only while NO images are being displayed. When I'm displaying an image on the second display, the shortcuts do nothing. If I select an alternative scene there, it switches right after I stop the image demonstration.

![PXL_20240113_180036172](https://github.com/sircharlo/meeting-media-manager/assets/80993061/c9622b00-d68c-478d-8740-4bdf8497fe8d)

**Comment by sircharlo** [comment]
This is by design. 

The "media" scene that is configured in the settings is "forced" whenever media is active. The scene picker buttons will only have an effect after the media is hidden. Otherwise, an accidental click on the scene picker while media is being shown on the screens at the KH would hide the media on Zoom and show a camera angle instead.

A scene can be configured for media, and one specifically for pictures (this picture scene can be set up in OBS with a PIP of the speaker, for example). These scenes are automatically excluded from the scene picker list in media playback mode, seeing as the scene picker only contains camera angles.

In reality, only one scene/angle is really needed for the stage view, that is, a wide view. Any other angle is considered "extra" but not required as per direction.

Hopefully that makes sense?

---

![image](https://github.com/sircharlo/meeting-media-manager/assets/1275666/7234bfc9-cd3f-4fc6-bd91-7a3e0a3cee97)

**Reply by ux-git** [reply]
Alright, thanks a lot for the reply now. I understand what the future was intended for – different camera angles. 

Although, in our case we successfully use these three main views configured in M3. Our additional scenes in OBS contain not the camera angles, but rather alternative "Picture mode" layouts for example: • speaker window in the top right corner • or top left corner • or bottom right corner, and so on). So I'm wondering that in this case, even if the scene picker will be clicked during the image presentation, it won't break the user flow, but will rather help to quickly change the scene without manually going into OBS and moving the PIP of the speaker to uncover an important part of today's illustration. 

But of course, it's hard to predict every use case and to optimize the app for this, but if you have any workaround for my scenario, I would be grateful to consider that. 

Thank you a lot for the answer and thank you for your wonderful application!
